fabfile/templates/docker-compose_kirin-beat.yml

```
@@ -5,7 +5,8 @@ services:
   kirin_beat:
     image: {{env.docker_image_kirin}}:{{env.current_docker_tag}}
     restart: unless-stopped
-    command: sh -c "rm -f /tmp/kirin-celerybeat.pid ; celery beat -A kirin.tasks.celery --pidfile=/tmp/kirin-celerybeat.pid"
+    hostname: {{env.name}}_{{env.docker_hostname_string}}_kirin_beat
+    command: sh -c "rm -f /tmp/kirin-celerybeat.pid ; {% if env.new_relic_key %}newrelic-admin run-program {% endif %}celery beat -A kirin.tasks.celery --pidfile=/tmp/kirin-celerybeat.pid"
```

To be set only once on top of each file then re-used:
```jinja2
{% if env.new_relic_key %}
{% set cmd_wrapper = 'newrelic-admin run-program' %}
{% else %}
{% set cmd_wrapper = '' %}
{% endif %}
```

```suggestion
    command: sh -c "rm -f /tmp/kirin-celerybeat.pid ; {{cmd_wrapper}} celery beat -A kirin.tasks.celery --pidfile=/tmp/kirin-celerybeat.pid"
```